src/mongocxx/test/collection.cpp

```diff
@@ -678,9 +713,9 @@ TEST_CASE("CRUD functionality", "[driver::collection]") {
         bulk.append(model::insert_one{std::move(doc3)});
         bulk.append(model::insert_one{std::move(doc4)});
 
-        distinct_coll.bulk_write(bulk);
+        coll.bulk_write(bulk);
```

Comment:
All of the sub-tests in the "CRUD functionality" test case use `coll`, except for tests that explicitly require multiple collections.  This change makes the "distinct test" consistent with the rest of the test cases.